README.md

```diff
@@ -1 +1,98 @@
-# House-Rental-API
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# House Rental
+
+This project an API  for the reservation of houses. If you're planning a special day or event, or if you want to spend time with your family, this is the website for you.You can book a house in any city on the planet. 
+Main features of this website are:
+
+- Users can see details of all the Houses.
+- Users can reserve a house for a special day and in the specific city.
+- Admin users can add and dlete houses.
+
+## Built With
+
+- Programming Languages: HTML, CSS, JavaScript.
+- Frameworks: React, Redux and Bootstrap.
+- Code Editor: VS Code.
+- Technologies: Git, Gitflow, Linters, Github, PostgreSql, Rspec, Capybara, Robocop
+
+## Link to front-end App 
+- Gitgub link [click](https://github.com/Sanja969/house-rental)
+- Heroku link [click](https://x-house.herokuapp.com)
+
+
+## Live Demo
+[Live Demo Link](https://x-house.herokuapp.com/)
+
+
+![Screen Shot 2022-11-09 at 11 44 04 AM](https://user-images.githubusercontent.com/102184682/200786497-eb5c8175-483a-4530-820c-4a64ed946fe8.png)
+
+
+- Kanban Board [link](https://github.com/users/Sanja969/projects/7)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- A web browser like Google Chrome.
+- A code editor like Visual Studio Code with Git and nodejs.
+- Have `visual-studio code` or any other code editor installed on your computer.
+
+### Setup
+
+- In order to get a copy of this project you need to download it from [link](https://github.com/Sanja969/house-rental-api.git)
+- Extract the zipped file and open it in your code editor
+
+### Install and start
+
+- Run the command bellow in your terminal to get all required files
+
+```
+npm install
+```
+
+```
+npm start
+```
```

Comment:
- [ ] These instructions correspond to the frontend app.  Please, kindly add detailed instructions on running your API project.  

- [ ] Please, don't forget a "tests" section to instruct your users about how to run the tests.

## Tips and extra help

In the "tests" section, since all your tests are rswag, the single command `rspec` won't work 

![image](https://user-images.githubusercontent.com/57421823/200818460-f1cd14a5-550c-4781-931b-79b66b603731.png)

Instead of that command, I suggest telling your users to use this one:

```
rake rswag
```

Result:

![image](https://user-images.githubusercontent.com/57421823/200818812-392c077e-0508-4d1d-a153-6e70c1d66855.png)

That way, your users will be able to run your app smoothly, and also the tests without failed examples, every time.  💪